knowledge-base/how-to-use-kendodatetimepicker-for-parameter-editors-in-the-angular-report-viewer.md

```diff
@@ -0,0 +1,120 @@
+---
+title: How to Use KendoDateTimePicker for Parameter Editors in the Angular Report Viewer
+description: See how to use the KendoDateTimePicker for selecting date and time for the report parameter by creating custom parameter editor in the Angular Report Viewer
+type: how-to
+page_title: Using KendoDateTimePicker in Angular Report Viewer
+slug: how-to-use-kendodatetimepicker-for-parameter-editors-in-the-angular-report-viewer
+position: 
+tags: angular, reportviewer, kendodatetimepicker
+ticketid: 1543447
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Project Type</td>
+			<td>Angular</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>Angular Report Viewer</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+This article describes how to use the [KendoDateTimePicker](https://docs.telerik.com/kendo-ui/api/javascript/ui/datetimepicker) as a custom parameter editor
+in [Angular Report Viewer](../angular-report-viewer). This approach allows the selection of date and time for the **DateTime** [report parameters](../designing-reports-parameters).
```

Comment:
I think it is better to use the new links:
../using-reports-in-applications/display-reports-in-applications/web-application/angular-report-viewer/angular-report-viewer-overview

../designing-reports/connecting-to-data/report-parameters/overview